include/json/value.h

```diff
@@ -89,9 +76,17 @@ class JSON_API LogicError : public Exception {
 #endif
 
 /// used internally
-JSONCPP_NORETURN void throwRuntimeError(String const& msg);
-/// used internally
-JSONCPP_NORETURN void throwLogicError(String const& msg);
+#if JSON_USE_EXCEPTION
+#define THROW_RUNTIME_ERROR(msg_ref)                                           \
+  { throw RuntimeError(msg_ref); }
+#define THROW_LOGIC_ERROR(msg_ref)                                             \
+  { throw LogicError(msg_ref); }
+#else
+#define THROW_RUNTIME_ERROR(msg_ref)                                           \
```

Comment:
It seems like abort() should be called in these macros for equivalence with the existing throwRuntimeError/throwLogicError?

For example JSON_ASSERT above will no longer abort() with this change.